includes/class-wc-stripe-order-handler.php

```diff
@@ -259,6 +259,10 @@ public function capture_payment( $order_id ) {
 						$net = ! empty( $result->balance_transaction->net ) ? WC_Stripe_Helper::format_balance_fee( $result->balance_transaction, 'net' ) : 0;
 						WC_Stripe_Helper::update_stripe_fee( $order, $fee );
 						WC_Stripe_Helper::update_stripe_net( $order, $net );
+
+						// Store currency stripe.
+						$currency = ! empty( $result->balance_transaction->currency ) ? strtoupper( $result->balance_transaction->currency ) : null;
+						WC_Stripe_Helper::update_stripe_currency( $order, $currency );
```

Comment:
wouldn't we only want to call this if `$currency` is not `null`? i didn't follow the full flow through the code so i may be misreading

Reply:
So we want the meta to save whether it is set or not.